sphinx/themes/classic/layout.html

```diff
@@ -15,3 +15,9 @@
     <script src="{{ pathto('_static/sidebar.js', 1) }}"></script>
     {% endif %}
 {%- endblock %}
+
+{%- block sidebarextra %}{% if theme_collapsiblesidebar|tobool %}
+<div id="sidebarbutton" title="{{ _('Collapse sidebar') }}">
+<span>{{ '»' if theme_rightsidebar|tobool else '«' }}</span>
+</div>
+{% endif %}{% endblock %}
```

Comment:
Hmm... this looks unrelated -- likely from separate work to add a right sidebar to the classic theme?